server/resources/resource_management.go

```diff
@@ -364,7 +364,7 @@ func CreateResource(
 		dataMap["tenant_id"] = context["tenant_id"]
 	}
 
-	if tenantID, ok := dataMap["tenant_id"]; ok {
+	if tenantID, ok := dataMap["tenant_id"]; ok && tenantID != nil {
```

Comment:
Now I see I was wrong.
On Jun 6, 2016 18:50, "Nachi Ueno" <notifications@github.com> wrote:

> In server/resources/resource_management.go
> <https://github.com/cloudwan/gohan/pull/240#discussion_r65925783>:
>
> > @@ -364,7 +364,7 @@ func CreateResource(
> >  		dataMap["tenant_id"] = context["tenant_id"]
> >  	}
> >
> > -	if tenantID, ok := dataMap["tenant_id"]; ok {
> > +	if tenantID, ok := dataMap["tenant_id"]; ok && tenantID != nil {
>
> tenantID could be nil even if we have key
> https://play.golang.org/p/a8hx_300QZ
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cloudwan/gohan/pull/240/files/f5b457a3ab451c822d7e10a945d1ca96ecf00b71#r65925783>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AC6QvBMqM3H5VKZ8bEQ4TI8SXv5vsBjyks5qJE_PgaJpZM4ItHGz>
> .
>